doc/stdlib/math.rst

```diff
@@ -1396,47 +1396,53 @@ Mathematical Functions
 
    Compute the polygamma function of order ``m`` of argument ``x`` (the ``(m+1)th`` derivative of the logarithm of ``gamma(x)``\ )
 
-.. function:: airy(k,x)
+.. function:: airyai(x)
```

Comment:
They do: I added `function airyai end` between the docstring and the Complex128 method.